frontend/public/module/k8s/index.ts

```diff
@@ -741,3 +741,9 @@ export type EventKind = K8sResourceKind & {
     host?: string;
   };
 };
+
+export type CD = {
```

Comment:
If i look correctly, this is used in `kubevirt-plugin` only. If so, please move it there